db/dataset_eval.py

```diff
@@ -261,16 +271,16 @@ def add_dataset_eval_set(connection, data):
     return snapshot_id
 
 
-def _create_job(connection, dataset_id, normalize, filter_type=None):
+def _create_job(connection, dataset_id, normalize, eval_location, filter_type=None):
```

Comment:
Maybe it would be nice to check if value in `eval_location` is valid before trying to insert. Like it's done with a `filter_type`.